languoids/tree/grea1241/md.ini

```diff
@@ -6,9 +6,10 @@ level = family
 [classification]
 sub = **hh:hv:Manoharan:Andamanese**
 familyrefs = 
-	**hh:hv:Abbi:Andamanese**
+	**hh:hv:Abbi:Andamanese** **hh:typ:ComrieZamponi:Great-Andamanese**
```

Comment:
Hm. If `set` receives a `list` as value, it should format the value such that each item is on a separate line. This is done by code in `clldutils.inifile`. Considering that the first value is on a new line, I'd guess it was created by
```python
 l.cfg.set('classification', 'familyrefs', ["**hh:hv:Abbi:Andamanese** **hh:typ:ComrieZamponi:Great-Andamanese**"])
```